Tests/test_imagecms.py

```diff
@@ -336,25 +337,13 @@ def test_extended_information() -> None:
     p = o.profile
 
     def assert_truncated_tuple_equal(
-        tup1: tuple[tuple[float, float, float], ...] | tuple[float],
-        tup2: (
-            tuple[tuple[tuple[float, float, float], ...], ...]
-            | tuple[tuple[float, float, float], ...]
-            | tuple[float]
-        ),
-        digits: int = 10,
+        tup1: tuple[Any, ...], tup2: tuple[Any, ...], digits: int = 10
     ) -> None:
         # Helper function to reduce precision of tuples of floats
         # recursively and then check equality.
         power = 10**digits
 
-        def truncate_tuple(
-            tuple_or_float: (
-                tuple[tuple[tuple[float, float, float], ...], ...]
-                | tuple[tuple[float, float, float], ...]
-                | tuple[float, ...]
-            )
-        ) -> tuple[tuple[float, ...], ...]:
+        def truncate_tuple(tuple_or_float: tuple[Any, ...]) -> tuple[Any, ...]:
```

Comment:
This parameter is called `tuple_or_float` but typed only as a `tuple`. 

Do we ever call it as a `float`? If not, shall we rename it?

Reply:
I've pushed a commit to rename it to `tuple_value`.